scene/debugger/script_debugger_remote.cpp

```diff
@@ -430,100 +433,86 @@ void ScriptDebuggerRemote::_err_handler(void *ud, const char *p_func, const char
 
 bool ScriptDebuggerRemote::_parse_live_edit(const Array &p_command) {
 
+#ifdef DEBUG_ENABLED
+
 	String cmdstr = p_command[0];
-	if (!live_edit_funcs || !cmdstr.begins_with("live_"))
```

Comment:
This is really nice clean up

Reply:
I had to make the debugger a friend of `SceneTree` for this but I think this is the kind of scenarios where it should be used. However, a question I wanted to ask is whether all _live_edit functions couldn't be moved to the remote debugger? They all use the public API as far as I could see and have to do with debugging. And keeping the `SceneTree` as compcat as possible might be worth the effort.